inland-empire-software-development/main
# Update mission and goal component to pull data from IESD API

Submission information

- [ ] Testing steps have been added to this issue or the PR for this issue
- [ ] Submission has been tested in all supported browsers

New Feature Submissions:
- [ ] Code gone through a code review by at least one other person
- [ ] Code has been locally linted yarn lint


Steps to test:
- Checkout the mission-goal-api-update branch `git checkout mission-goal-api-update`
- Run `yarn install && yarn dev` 
- Visit `localhost:3000`
- Visit the main page for the application
- Scroll down to the mission section - check if text has been populated.
- Scroll down to the goal section - check if text has been populated. 
- You can also check in the Network tab, make sure that we're pulling from the following: https://api.iesd.com/wp-json/iesd/api/settings?set=organization&name=goal AND https://api.iesd.com/wp-json/iesd/api/settings?set=organization&name=mission
